Updater/Utils.cs

```diff
@@ -85,7 +85,8 @@ public static string SerializeObject<T>(T obj)
             ISerializer serializer = new Serializer();
             return serializer.Serialize(obj);
         }
-        catch (Exception ex) { 
+        catch (Exception ex)
+        {
             Console.WriteLine(ex.ToString());
```

Comment:
Don't use Console statements, use Trace

Reply:
Lol I didn't do it
Anyways changed them